lambda.tf

```diff
@@ -2,27 +2,22 @@ locals {
   lambda_function_name = "${var.global_prefix}-lambda"
 }
 
-locals {
-  enabled_notifiers = join(",", [
-    for value in local.notifiers : upper(value.name)
-    if value.enabled
-  ])
-  parameters_map_list = tolist([
-    for value in local.notifiers : {
-      for param_name, param_value in value.parameters :
-      "${upper(value.name)}_${param_name}" => param_value
-    }
-    if value.enabled
-  ])
 
-  parameters_map = merge(local.parameters_map_list...)
+resource "local_sensitive_file" "notifier_parameters" {
+  content         = jsonencode(local.ggcanary_lambda_parameters)
+  filename        = "lambda/lambda_py/data/ggcanary_lambda_parameters.json"
```

Comment:
Is this file saved locally or only in the lambda ?
I think it is very prone to errors if the file is created and stored locally. In particular in the case it is overwritten.

Reply:
Yes the file is saved locally. I did not find a way to pass the file directly (because of the way https://registry.terraform.io/modules/terraform-aws-modules/lambda/aws/latest works). 

However the file is entirely re-written each time `terraform plan` is called, so there should be no risk associated with manual changes to the file.

Reply:
The biggest issue I find with the local file is in fact that it will be overwritten each time, triggering changes to be applied (lambda redeploy) even without configuration change.

Reply:
I think it is not very good to write the secrets in a local file.